firmware/arduino/chessboard/chessboard.ino

```diff
@@ -44,13 +44,17 @@ uint8_t yMotor[6] = {Y_MOTOR_STEP, Y_MOTOR_DIR, Y_MOTOR_MS1, Y_MOTOR_MS2, Y_AXIS
 #define MILLIMETERS_PER_UNITSPACE 32
 #define STEPS_PER_MILLIMETER 5  // Whole steps per millimeter
 #define HOME_CALIBRATION_OFFSET 100
+#define TOTAL_UNITSPACES 24
 
 // Number of whole steps per unit space
 int stepsPerUnitSpace;
 
 // currentX and currentY measure distance from the origin (bottom left corner of the board) in eighth steps
 int currentX, currentY;
```

Comment:
To make these names symmetrical to the updated name for `posExtreme`, can you please make these `currentPosX` and `currentPosY`?

Reply:
"pos" may not be indicative enough for what each of the different meanings are

Reply:
please make this `currPositionX` and `currPositionY`